firebase/src/core/data/gateways/firebase-app.ts

```diff
@@ -1,5 +1,18 @@
 import * as admin from "firebase-admin";
+import { EnvGateway } from "./env-gateway";
 
-// TODO: Properly initialize this, taking into consideration the environment
-process.env["FIRESTORE_EMULATOR_HOST"] = "localhost:8080";
-export const app = admin.initializeApp({ projectId: "test-rpoj" });
+export function initializeFirebase(env: EnvGateway): admin.app.App {
+  if (env.isLocalDevelopment) {
+    process.env["FIRESTORE_EMULATOR_HOST"] = "localhost:8080";
+  }
+
+  const serviceAccount = env.firebaseServiceAccount;
+  return admin.initializeApp({
+    projectId: serviceAccount.projectId,
+    credential: admin.credential.cert({
+      clientEmail: serviceAccount.clientEmail,
+      projectId: serviceAccount.projectId,
+      privateKey: serviceAccount.privateKey,
+    }),
```

Comment:
The `credentials` initialization it's not required when running the `functions` environment. But when running scripts locally, we need to properly authenticate with Firebase. To support both scenarios, I suggest to we always initialize with Firebase Service Account metadata